projects/core/src/lib/types/mask-processors.ts

```diff
@@ -0,0 +1,8 @@
+import {ElementState} from './element-state';
+
+export type MaskPreprocessor = (_: {elementState: ElementState; data?: string}) => {
+    elementState: ElementState;
+    data?: string;
+};
+
+export type MaskPostprocessor = (elementState: ElementState) => ElementState;
```

Comment:
This might need `rawState` as well, to know what state was before mask did its job.

Reply:
https://github.com/Tinkoff/maskito/pull/47